molo/core/api/endpoints.py

```diff
@@ -71,6 +71,11 @@ def get_queryset(self):
         queryset = queryset.descendant_of(
             request.site.root_page, inclusive=True)
 
+        # Enable filtering by navigation tags
+        if model == ArticlePage and 'nav_tags__tag' in request.GET:
+            queryset = queryset.filter(
+                nav_tags__tag=request.GET['nav_tags__tag'])
+
```

Comment:
@KaitCrawford I think we need validation for this var `request.GET['nav_tags__tag']` perhaps using django-filters